### ISSUE TITLE
Cannot escape from "The Ebon Hold" when on the quest "The Lich King's Command"

### ISSUE THREAD

**Issue by BobsYurUnkle** [dalaranwow/dalaran-wow#5101]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
WHAT:  I am currently using a DeathKnight character and going through the usual class based quests.  I have just started the quest "The Lich King's Command" and before doing the quest I decided to sell some items in my inventory to free up space.  The small camp around the Lich King was unpopulated, however, and as a result I could not sell items there.  I decided therefor to travel up to "Acherus, The Ebon Hold" and sell my unwanted items to a merchant there.  Unfortunately no merchants were there (in fact no NPC's at all) and now I am unable to escape that area.

WHO:  Barog, level 58 Undead Death Knight

WHERE:  The Ebon Hold

WHY:  Because I wanted to sell unwanted items to free up inventory space

WHEN:  Tuesday, Nov 21/2017, 18:35 PST

**Expected behaviour**: Tell us what should happen instead
I would expect that either
    1) a player would not be able to go up to "The Ebon Hold" OR
    2) there would be a way to leave "The Ebon Hold"

**Steps to reproduce the problem**:

1.  create a Death Knight character (I used an undead for the race but I don't think it matters)
2.  finish all quests until you get to "The Lich King's Command"
3.  take one of the flying undead gryphons up to "The Ebon Hold"
4.  enjoy not being able to leave (kind of like the Hotel California only less fun)

**Include proofs for this behaviour**
http://www.wowhead.com/forums&topic=251696/cant-leave-acherus-the-ebon-hold
This is a similar situation except that the character mentioned in the above post used the "Death Gate" spell to enter The Ebon Hold and I used the undead gryphons from the encampment below.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

"The Lich King's Command" Quest
https://wowgaming.altervista.org/aowow/?quest=12800

The Eastern Plaguelands
https://wowgaming.altervista.org/aowow/?zone=139

I don't know what other information is required, sorry.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @BobsYurUnkle.
I've tried to reproduce the issue on PTR by manually going though the DK's quest chain and when I got to the quest "The Lich King's Command" I took the grifon to the hold and I was able to see all creatures and I could leave it the normal way down.
![image](https://user-images.githubusercontent.com/6797611/33119371-75402630-cf77-11e7-866e-237bd6e30786.png)

Did you used the unstuck feature by any chance to get out of the proper phase?

**Comment by BobsYurUnkle** [comment]
Hey!

Thanks for the quick response. That is indeed interesting. I had actually tried re-logging a few times to see if that would fix the issue but there want any change. I wonder what went wrong for me. 

No I didn’t even know there is an unstuck feature. That does sound pretty useful though. What is it and how can I use it?

**Comment by TeridanDW** [comment]
> No I didn’t even know there is an unstuck feature. That does sound pretty useful though. What is it and how can I use it?

In this case it won't help, best option is to create a ticking ingame and wait for a GM to assist you (note: be online as long as possible on your DK).
Unstuck feature:
![untitled](https://user-images.githubusercontent.com/6797611/33123358-a4e3d674-cf82-11e7-8ec6-9e894d7b78e8.png)

**Comment by BobsYurUnkle** [comment]
How do I create a ticket in-game??

**Comment by TeridanDW** [comment]
Press green then red and write as detailed as possible what is your issue.
![untitled](https://user-images.githubusercontent.com/6797611/33130841-562775c0-cf9d-11e7-88ed-ae232f39d121.png)

**Comment by BobsYurUnkle** [comment]
Thank you very much. I logged in and created an in-game ticket now.
